proto/strings.proto

```diff
@@ -3,19 +3,39 @@ syntax = "proto3";
 package strings;
 
 service StringsExtractor {
-   rpc ExtractStrings (StringsRequest) returns (StringsReply) {}
+	rpc ExtractStrings (StringsRequest) returns (StringsReply) {}
 }
 
 message StringsRequest {
-   // ID of file.
-   string id = 1;
+	// Binary ID.
+	string bin_id = 1;
 }
 
 message StringInfo {
+	// File offset to start of string.
 	uint64 location = 1;
+	// String in UTF-8 encoding.
```

Comment:
Also, refresh my memory, UTF-8 is capable of presenting any code point also present in UTF-16 and UTF-32, _right_? So there is a lossless conversion to UTF-8 from all other major formats?

Reply:
Indeed. UTF-8 can represent any Unicode code point.